gammapy/irf/psf/tests/test_map.py

```diff
@@ -567,6 +567,11 @@ def test_psf_map_reco(tmpdir):
 
 @requires_data()
 def test_psf_map_reco_hawc():
+    # There are overlapping GTIs in this HAWC dataset
+    from gammapy.data import utils
+
+    utils.CHECK_OVERLAPPING_TI = False
+
```

Comment:
Here the test is only about the PSF, why do we need a gti check here ?

Reply:
good point! fixed..